components/brave_wallet/browser/solana_tx_manager.cc

```diff
@@ -145,13 +145,14 @@ void SolanaTxManager::OnGetLatestBlockhashHardware(
   meta->tx()->message()->set_last_valid_block_height(last_valid_block_height);
   tx_state_manager_->AddOrUpdateTx(*meta);
 
-  auto message_bytes = meta->tx()->message()->Serialize(nullptr);
-  if (!message_bytes) {
+  std::vector<uint8_t> message_bytes =
+      meta->tx()->GetSerializedMessage()->first;
```

Comment:
Should first get the pair and check if it's absl::nullopt before accessing message_bytes in the pair.